### ISSUE TITLE
Error: Process exited with no output, code: 0

### ISSUE THREAD

**Issue by roryjf** [AtomLinter/linter-clang#177]
I'm getting the following error when I save a file which has no warnings or errors as of the newest update that was just committed:

```
Error: Process exited with no output, code: 0
    at ChildProcess.<anonymous> (/Users/roryf/.atom/packages/linter-clang/node_modules/sb-exec/lib/index.js:67:20)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

**Comment by NanerLee** [comment]
I also meet same problem

**Comment by mortal123** [comment]
same here

**Comment by mortal123** [comment]
Seems that enable the verbose debug may stop these errors temporarily.

**Comment by luyiming** [comment]
same here
clang version: 3.8.0-2ubuntu4 (tags/RELEASE_380/final)
Atom: 1.12.7

**Comment by Paalon** [comment]
I have came across nearly same error.
Error: Process exited with no output, code: 0
    at ChildProcess.<anonymous> (/Users/myname/.atom/packages/linter-clang/node_modules/sb-exec/lib/index.js:67:20)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)

OS: Mac OSX 10.11.6
clang --version:
Apple LLVM version 8.0.0 (clang-800.0.42.1)
Target: x86_64-apple-darwin15.6.0
Atom: 1.13.0 x64

**Comment by Paalon** [comment]
It's only showed there no syntax mistake.

**Comment by diraimondo** [comment]
@mortal123 could you argue? what's the setting should be tweaked in order to workaround the annoying message?

**Comment by Paalon** [comment]
@diraimondo linter-clang Settings -> Verbose Debug
check it.

**Comment by diraimondo** [comment]
Thank you: I disabled linter-clang package for the nasty bug but, after re-enabling it, it was necessary to restart Atom to see the settings (all of them!)...

**Comment by Arcanemagus** [comment]
Odd, I wonder why this didn't show up in the CI tests, I'll push out a fix shortly to ignore the exit code. (Even stranger that the verbose debug option does anything related to this)

**Comment by Arcanemagus** [comment]
Looks like the specs never tested for a valid file 😞. Fixing that in the same PR that will fix this bug...

**Comment by Arcanemagus** [comment]
Fix is published in v3.4.6, let me know if there are any further issues 😉.
